src/collection-view.js

```diff
@@ -499,6 +503,19 @@ Marionette.CollectionView = Marionette.View.extend({
     return childViews;
   },
 
+  // Return a function for determining whether or not to display a given model
+  // The function must match the interface (Model) -> Boolean
+  getFilter: function() {
+    return this.getOption('filter');
+  },
+
+  // Return true if the given child should be shown
+  // Return false otherwise
+  testChild: function (child) {
+    var filter = this.getFilter();
```

Comment:
Following the pattern set by `getEmptyView` and `getViewComparator`, I wrapped `getOption('filter')` in the method `getFilter`.